src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -15,17 +15,34 @@
 import logging
 import os
 
+import mlio
+import numpy as np
+import pandas as pd
+import pyarrow.parquet as pq
 import xgboost as xgb
 
 from sagemaker_algorithm_toolkit import exceptions as exc
 from sagemaker_containers import _content_types
 from sagemaker_xgboost_container.constants import xgb_content_types
 
+from mlio.integ.numpy import as_numpy
+from mlio.integ.arrow import as_arrow_file
+
+BATCH_SIZE = 4000
```

Comment:
Where does this come from?

Reply:
I arbitrarily chose a value, since we need to specify a batch size for the mlio data readers. I have not experimented yet if memory consumption is affected by this value.